examples/plugins/alloy/AlloyPlugin.java

```diff
@@ -13,17 +13,16 @@
 
 
```

Comment:
Would you be able to write a short description of how the Alloy plugin translates the Substance program into an Alloy description? This would be helpful for the paper.